website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -3815,6 +3863,14 @@ The following
 purchasesByType.apply(Select.fieldNames("purchases{}.userId"));
 {{< /highlight >}}
 
+{{< paragraph class="language-py" >}}
+Support for Nested fields hasn't been developed for python SDK yet
+{{< /paragraph >}}
+
+{{< paragraph class="language-go" >}}
+Support for Nested fields hasn't been developed for python SDK yet
```

Comment:
```suggestion
{{< paragraph class="language-py" >}}
Support for Map fields hasn't been developed for the Python SDK yet.
{{< /paragraph >}}

{{< paragraph class="language-go" >}}
Support for Map fields hasn't been developed for the Go SDK yet.
```

Reply:
nit: in general can we make these paragraphs "Support for X hasn't been developed for the {Python,Go} SDK yet."

Reply:
ohh thanks I'll keep that in mind